### ISSUE TITLE
Not only primary seed color

### ISSUE THREAD

**Issue by T8RIN** [jordond/MaterialKolor#125]
Is it possible to use this library, to create schemes based on 4 base colors? Primary, Secondary, Tertiary and Surface ? I mean it would be useful to have some func

```kotlin
dynamicColorSceme(
      primary: Color, 
      secondary: Color? = null,
      tertiary: Color? = null, 
      surface: Color? = null
)
```

And if value is null then it will be calculated from primary as always

**Comment by jordond** [comment]
I suppose you could create your own `DynamicScheme` that passes other colors as the secondary and tertiary palette.

But as of right now I have no plans to implement that, maybe in the future. Feel free to look into it an open up a PR!

**Comment by T8RIN** [comment]
Okay 👀

**Comment by T8RIN** [comment]
This can be used in my use case -> https://github.com/jordond/MaterialKolor/blob/1a328f3277ee2150ebcad471b6860c5a2ed73889/material-color-utilities/src/commonMain/kotlin/com/materialkolor/scheme/DynamicScheme.kt

**Comment by jordond** [comment]
@T8RIN Could you elaborate a bit more on your use-case for this proposed feature?

**Comment by Nek-12** [comment]
I have the same use case and can elaborate. Our app has multiple brand colors - primary, secondary and tertiary, in addition to the custom error color. Configuring them was available with the old google's builder which is abandoned now. We can't preserve our brand identity without being able to configure these 4 colors, so that's the main reason we're not using MK right now

**Comment by jordond** [comment]
Implemented in #213, released in 2.0.0

Try it out at [materialkolor.com](https://materialkolor.com)!
